release.sh

```diff
@@ -15,7 +15,7 @@ then
   echo "spotify/web-scripts: Configuring npm for publishing..."
   echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
   echo "spotify/web-scripts: Attempting publish..."
-  npx lerna publish --yes --conventional-commits --create-release=github --registry=https://registry.npmjs.org
+  npx lerna publish --yes --force-publish --conventional-commits --create-release=github --registry=https://registry.npmjs.org
```

Comment:
I think this warrants explanation (copy/paste your PR write-up somewhere in here)

Reply:
great suggestion! I will append the explanation to the script as a comment